Sources/XiCliCore/CommandLineTool.swift

```diff
@@ -70,6 +80,10 @@ public final class CommandLineTool {
         }
         
         let pathString = filePath.path
+
+        guard !pathIsDirectory(pathString) else {
```

Comment:
nit: Wouldn't it be more readable if we didn't inverse that logic?
What do you think of something like:
```
if pathIsDirectory(pathString) else {
    throw CliError.pathIsDirectory
}
```
? 😄

Reply:
Doesn't this mean throwing if the path is not a directory? Shouldn't this be 
`if pathIsDirectory(pathString) {
    throw CliError.pathIsDirectory
}`
?

Reply:
You’re right, my code won’t even compile 😅

Reply:
:) Though I think your comment makes sense, as not inverting `guard` logic seems to make it easier to read. I'll change this up a bit.